### PR TITLE
Fix mapping for sort_field

### LINKED ISSUE #3368: [dor-services-app/stage] Cocina::RoundtripValidationError: Roundtripping of descriptive metadata unsuccessful. Expected {"title":[{"structuredValue":[{"structuredValue":[],"parallelValue":[],"groupedValue":[],"value":"FFIEC HMDA raw data 2003","type":"main title","identifier":[],"note":[],"appliesTo":[]},{"structuredValue":[],"parallelValue":[],"groupedValue":[],"value":"part 17","type":"part number","identifier":[],"note":[],"appliesTo":[]},{"structuredValue":[],"parallelValue":[],"groupedValue":[],"val...

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made? 🤔
Fixes #3368
The prior mapping was causing a roundtrip validation error in DSA

## How was this change tested? 🤨
Added an integration test.

### CODE REVIEW

File: spec/forms/serials_form_spec.rb [hunk 1]
```diff
@@ -30,13 +30,13 @@
...
                 { value: '7', type: 'part number' },
                 { value: 'samurai', type: 'part name' }
-              ],
-              note: [
-                { value: '1990', type: 'date/sequential designation' }
               ]
             }
           ],
-          'purl' => "https://purl.stanford.edu/#{druid.delete_prefix('druid:')}"
+          note: [
```
Comment (ndushay) [inline spec/forms/serials_form_spec.rb:36]: is it intended that you changed the level at which the note is present in the cocina object?

**Review by ndushay — CHANGES_REQUESTED** [review]
I'm not sure this is going in the correct direction.  See comment I put on #3368.  Please confer with Arcadia, and please can we have a better description of the problem?  how this relates to a sort field is beyond me.

File: spec/forms/serials_form_spec.rb (hunk 1 above)
Comment (jcoyne) [reply spec/forms/serials_form_spec.rb:36]: Yes, that's the change.

Comment (jcoyne) [reply spec/forms/serials_form_spec.rb:36]: Screen Shot 2022-03-31 at 2 24 34 PM

**Review by ndushay — APPROVED** [review]

**Pull request merged**